src/bci_build/package.py

```diff
@@ -1235,6 +1274,100 @@ def generate_disk_size_constraints(size_gb: int) -> str:
 """
 
 
+BASALT_BASE = OsContainer(
+    name="base",
+    pretty_name="Base",
+    package_name="base-image",
+    build_recipe_type=BuildType.KIWI,
+    from_image=None,
+    os_version=OsVersion.BASALT,
+    is_latest=True,
+    package_list=[
+        Package(name=pkg_name, pkg_type=PackageType.BOOTSTRAP)
+        for pkg_name in (
+            "aaa_base",
```

Comment:
IMO there should be a comment listing the reason for most of them. https://build.opensuse.org/package/view_file/openSUSE:Factory/opensuse-tumbleweed-image/opensuse-tumbleweed-image.changes?expand=1 mentions most of those with ticket number.

Reply:
good point, when we lose the existing `.changes` file via the generator then keeping that information around would be useful..